store/server/features/leads/lead-types/mutation.ts

```
@@ -0,0 +1,211 @@
+    const headers = {
+      tenantId: tenantId,
+      Authorization: `Bearer ${token}`,
+    };
+
+    // Add tenant ID to the payload for backend validation
+    const payloadWithTenant = {
+      ...data,
+      tenantId: tenantId,
+    };
+
+    console.log('payloadWithTenant', payloadWithTenant);
```

ESLint

[error] Unexpected console statement.

(no-console)